autometrics/src/otel_push_exporter.rs

```diff
@@ -115,6 +95,26 @@ pub fn init_grpc_with_timeout_period(
         .map(OtelMeterProvider)
 }
 
+/// returns timeout and period from their respective environment variables
+/// or the default, if they are not set or set to an invalid value
+fn timeout_and_period_from_env_or_default() -> (Duration, Duration) {
+    let timeout = Duration::from_secs(
+        std::env::var_os("OTEL_METRIC_EXPORT_TIMEOUT")
+            .and_then(|os_string| os_string.to_str())
+            .and_then(|str| str.parse().ok())
+            .unwrap_or(OTEL_EXPORTER_OTLP_TIMEOUT_DEFAULT),
+    );
+
+    let period = Duration::from_secs(
+        std::env::var_os("OTEL_METRIC_EXPORT_INTERVAL")
+            .and_then(|os_string| os_string.to_str())
+            .and_then(|str| str.parse().ok())
+            .unwrap_or(60),
```

Comment:
It would be nice to name the constant so it’s easier to find if we want to change the values later